atom/browser/native_window_mac.mm

```diff
@@ -1358,22 +1358,22 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
     // they are available in the minimum SDK version
     if (type == "selection") {
       // NSVisualEffectMaterialSelection
-      vibrancyType = (NSVisualEffectMaterial) 4;
+      vibrancyType = static_cast<NSVisualEffectMaterial>( 4);
```

Comment:
The leading space in front of the number here (and below) seems like an oversight.